.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -80,9 +80,9 @@ If user exposed functionality or configuration variables are added/changed:
 
 If the code communicates with devices, web services, or third-party tools:
 
-- [ ] The [manifest file][manifest-docs] has all fields filled out correctly.  
+- [ ] The [manifest file][manifest-docs] has all fields filled out correctly.
       Updated and included derived files by running: `python3 -m script.hassfest`.
-- [ ] New or updated dependencies have been added to `requirements_all.txt`.  
+- [ ] New or updated dependencies have been added to `requirements_all.txt`.
```

Comment:
The spaces here are needed, else it generates one big line.

Reply:
Maybe we should put the code in a real code block on a separate line so we don't need to manage the wrapping?

Reply:
Yeah, that is fine as well, as long as it doesn't become one line 👍